examples/react-todos/.meteor/versions

```diff
@@ -60,6 +59,7 @@ sha@1.0.3
 spacebars@1.0.6
 spacebars-compiler@1.0.6
 srp@1.0.3
+static-html@0.0.1
 templating@1.1.1
```

Comment:
It's definitely not running inside the app, because then `static-html` wouldn't work since you can't have two handlers for the same extension.